app/views/contexts/section_summary_context.py

```diff
@@ -148,3 +158,11 @@ def _add_link(self, summary, current_location, section, list_collector_block):
                 block_id=driving_question_block["id"],
                 return_to_summary=True,
             )
+
+    def _get_safe_page_title(self, title, title_has_placeholders=False):
+        if title_has_placeholders:
+            if "text_plural" in title:
+                title = title["text_plural"]["forms"]["other"]
```

Comment:
As the number is replaced with ... and other is always there, this seems the most logical one to use